app/facades/orders/new_facade.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Orders
+  class NewFacade
+    def initialize(**params)
+      @order = params[:order]
+    end
+
+    def order
+      @order ||= Order.new(order_meals: new_order_meals).decorate
+    end
+
+    def order_saved?
+      order.persisted?
+    end
+
+    def menu
+      @menu ||= Menu.last
```

Comment:
+1000 for memoization, -1488 for `Menu.last`.
At this point, I really hope I just missed something meaningful and `Menu.last` is a good thing.